r/R/feather.R

```diff
@@ -154,7 +154,15 @@ read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
     vars_select(names(reader), !!col_select)
   }
 
-  out <- reader$Read(columns)
+  out <- tryCatch(
+    reader$Read(columns),
+    error = function (e) {
+      if (grepl("Support for codec", conditionMessage(e))) {
+        msg <- "Unsupported compressed format: We suggest either setting the right environment variable to install binaries or setting LIBARROW_MINIMAL=false and then reinstall the package."
```

Comment:
1. Name the compression format
2. "setting the right environment variable" isn't helpful--which is the right one? (Hint: you know the codec from the error message)